tests/400-serlo-editor.ts

```diff
@@ -89,3 +109,124 @@ Scenario('Markdown list shortcut', async ({ I }) => {
 
   I.seeElement('ul[data-slate-node="element"]')
 })
+
+Scenario('Copy/cut/paste text', async ({ I }) => {
+  I.amOnPage('/entity/repository/add-revision/74888')
+
+  I.click('span[data-slate-node="text"]')
+
+  I.type(' ')
+
+  I.type('TESTTESTTEST')
+
+  I.see('TESTTESTTEST')
+
+  for (let i = 0; i < 12; i++) {
+    I.pressKey(['Shift', 'LeftArrow'])
+  }
+
+  I.pressKey(['Ctrl', 'C'])
+
+  I.pressKey('Backspace')
+
+  I.dontSee('TESTTESTTEST')
+
+  I.pressKey(['Ctrl', 'V'])
+
+  I.see('TESTTESTTEST')
+
+  for (let i = 0; i < 12; i++) {
+    I.pressKey('Backspace')
+  }
+
+  I.type('CUTCUTCUT')
+
+  I.see('CUTCUTCUT')
+
+  for (let i = 0; i < 9; i++) {
+    I.pressKey(['Shift', 'LeftArrow'])
+  }
+
+  I.pressKey(['Ctrl', 'X'])
+
+  I.dontSee('CUTCUTCUT')
+
+  I.pressKey(['Ctrl', 'V'])
+
+  I.see('CUTCUTCUT')
+})
+
+Scenario('Toggle on and off', async ({ I }) => {
+  I.amOnPage('/entity/create/Article/1377')
+
+  I.click('Füge ein Element hinzu')
+
+  I.pressKey('Backspace')
+
+  I.type('Some text')
+
+  I.see('Some text')
+
+  I.pressKey(['Ctrl', 'A'])
+
+  //Bold + Italic
+
+  I.pressKey(['Ctrl', 'I'])
+
+  I.seeElement({ css: 'em' }) //em = italic
+
+  I.pressKey(['Ctrl', 'I'])
+
+  I.dontSeeElement({ css: 'em' })
+
+  I.pressKey(['Ctrl', 'B'])
+
+  I.seeElement({ css: 'strong' })
+
+  I.pressKey(['Ctrl', 'B'])
+
+  I.dontSeeElement({ css: 'strong' })
+
+  //Create Link
+
+  //Use Hack because a id or name of link button is missing
+
+  I.pressKey(['Ctrl', 'K'])
+
+  I.type('https://serlo.org')
+
+  I.click('span[data-slate-string="true"]')
+
+  I.seeElement('button.serlo-tooltip-trigger.cursor-pointer:nth-child(3)')
+
+  I.click('Some text')
+
+  I.amOnPage('https://serlo.org')
```

Comment:
Can you maybe explain why you are leaving the editor (probably to reset content?)

You can use `I.amOnpage('/')` to make it independent of the domain